### DISCUSSION TITLE
getCacheEntry failed: ﻿{"$id":"1","innerException":null,"message":"Database pool DefaultPartitionPool is full. Raise the MaxDatabaseLimit or use a different pool

### DISCUSSION THREAD

**Discussion by gregnrobinson** [community/community#25791] [Actions]
<p>Github Actions is currently throwing database errors during my pipeline runs. I have run the same workflow templates running fine on different repos and it works but this specific repo is kind of borked I guess, or it’s physically located somewhere where there are some technical problems. Just thought I would share.</p>
<pre><code class="lang-bash">getCacheEntry failed: ﻿{"$id":"1","innerException":null,"message":"Database pool DefaultPartitionPool is full. Raise the MaxDatabaseLimit or use a different pool.","typeName":"Microsoft.TeamFoundation.Framework.Server.DatabasePoolFullException, Microsoft.TeamFoundation.Framework.Server","typeKey":"DatabasePoolFullException","errorCode":0,"eventId":3000}
</code></pre>
<p>the initial error was…</p>
<pre><code class="lang-bash">Error: getCacheEntry failed: { 'message': 'GitHub Actions is temporarily unavailable. Please visit https://www.githubstatus.com/ for the status of our services.', 'ref': 'Ref A: D891D4B3C8234FE380B94CCCB656B195 Ref B: BL2EDGE1718 Ref C: 2021-12-12T06:48:13Z' }
</code></pre>
<p>The issue seems out of my control…</p>
<p>Failed Runs: <a href="https://github.com/gregnrobinson/kube-secrets/runs/4498019768?check_suite_focus=true" class="inline-onebox" rel="noopener nofollow ugc">Update pre-commit-check.yaml · gregnrobinson/kube-secrets@3baa48c · GitHub</a></p>
<p>I don’t think the repo has been able to do one successful run yet.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/gregnrobinson/kube-secrets/runs/4498019768?check_suite_focus=true">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/gregnrobinson/kube-secrets/runs/4498019768?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/26353407/181093291-f2494be5-75b1-4279-bd9d-f9b3a3df96c5.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ea50ec29302575a0f9e7d84f234d2fce45b0d8e_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ea50ec29302575a0f9e7d84f234d2fce45b0d8e_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/e/6ea50ec29302575a0f9e7d84f234d2fce45b0d8e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/e/6ea50ec29302575a0f9e7d84f234d2fce45b0d8e_2_10x10.png"></div>

<h3><a href="https://github.com/gregnrobinson/kube-secrets/runs/4498019768?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">Update pre-commit-check.yaml · gregnrobinson/kube-secrets@3baa48c</a></h3>

  <p>A project that provides a simple interface to deploy Kubernetes secrets from a single configuration file. - Update pre-commit-check.yaml · gregnrobinson/kube-secrets@3baa48c</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by Hbbb** [comment]
<p>I’m seeing the same error with a different action.</p>
<pre><code class="lang-auto">Error: getCacheEntry failed: ﻿{"$id":"1","innerException":null,"message":"Database pool DefaultPartitionPool is full. Raise the MaxDatabaseLimit or use a different pool.","typeName":"Microsoft.TeamFoundation.Framework.Server.DatabasePoolFullException, Microsoft.TeamFoundation.Framework.Server","typeKey":"DatabasePoolFullException","errorCode":0,"eventId":3000}
</code></pre>
<p>The <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">Github Status Page</a> shows no active incidents on Actions.</p>

**Comment by dhadka** [comment, marked as answer]
<p>Hi, thanks for reporting this.  The issue should now be mitigated.  We’re tracking any followup at <a href="https://github.com/actions/cache/issues/698#issuecomment-992069658" class="inline-onebox" rel="noopener nofollow ugc">Cache creation failed · Issue #698 · actions/cache · GitHub</a></p>

**Comment by Hbbb** [comment]
<p>Thank you. That fixed the issue for me. :+1:</p>
